q2_phylogeny/citations.bib

```diff
@@ -10,3 +10,16 @@ @article{price2010fasttree
   publisher={Public Library of Science},
   doi={10.1371/journal.pone.0009490}
 }
+
+@article{Stamatakis2014raxml,
+author = {Stamatakis, Alexandros},
```

Comment:
Sorry, this is a little nitpicky, but do you think you could match the line indentation for 15-23 to match the citation entries above?

Reply:
Excellent point @ebolyen - I think I was coming from less of a "style" perspective and more of a "Bibtex PTSD" perspective - playing with poorly behaved parsers in the past has scarred me. If you think this is purely a style issue, then I am all on board with leaving things as-is (here, and in general)!

Reply:
Ditto, we'll get the worlds longest stacktrace (like 3 nested raise-froms) when something is actually wrong. Otherwise, this will all get massaged into a standard indentation, curly-brace-quoted, and latex-decoded format.

Reply:
Ahh, no problem. I am surprised my indentation OCD failed me in a big way here. :-)